common/src/test/java/org/apache/uniffle/common/RssShuffleUtilsTest.java

```diff
@@ -46,4 +48,27 @@ public void testCompression(int size) {
     assertArrayEquals(data, buffer2);
   }
 
+  @Test
+  public void testDestroyDirectByteBuffer() throws Exception{
+    int size = 10;
+    byte b = 1;
+    System.out.println(b);
```

Comment:
remove?

Reply:
agree

Reply:
I means that we remove 
```
System.out.println
```
I think this test is necessary. We shouldn't remove it.